pkg/netceptor/netceptor.go

```diff
@@ -432,6 +443,17 @@ func (s *Netceptor) MaxConnectionIdleTime() time.Duration {
 	return s.maxConnectionIdleTime
 }
 
+// Convert the connection type to a string.
+func (s *Netceptor) GetConnectionTypeAsString(connectionType byte) string {
+	// A byte can't be < 0 so we don't need to check the lower bounds
+	connTypeString, ok := ConnTypeStrings[connectionType]
+	if !ok {
+		connTypeString = UnknownConnTypeStr
```

Comment:
Curious, under which conditions would this happen?

Reply:
If someone added a new connection type constant but forgot to add an entry into the map. I was also thinking of some weird upgrade scenario where a newer version of receptor with a new connection type was introduced to an existing mesh.